src/components/InputField/InputFieldStyled.js

```diff
@@ -53,53 +56,70 @@ export const StyledInput = styled.input`
   padding-right: ${({ type }) => (type === 'password' ? '48px' : '16px')};
 
   @media screen and (min-width: 768px) {
-    background-color: ${({ $backgroundcolor }) => $backgroundcolor || 'var(--color-grey-4)'};
+    background-color: ${({ $backgroundcolor }) =>
+      $backgroundcolor || 'var(--color-grey-4)'};
   }
 `;
 
 export const StyledTextarea = styled(TextareaAutosize)`
   box-sizing: border-box;
   width: 100%;
   resize: none;
-  font-family: "Roboto", sans-serif;
+  font-family: 'Roboto', sans-serif;
   font-size: 18px;
   line-height: 21.6px;
   overflow: auto !important;
   height: 56px;
   border-radius: 8px;
   border: 1px solid;
   padding: 18px 16px;
-  border-color: ${({ disabled }) => (disabled ? 'transparent' : getNotificationColor)};
+  border-color: ${({ disabled }) =>
+    disabled ? 'transparent' : getNotificationColor};
 
   &:hover {
-    border: 1px solid ${({ disabled }) => (disabled ? 'transparent' : 'var(--color-dark)')};
+    border: 1px solid
+      ${({ disabled }) => (disabled ? 'transparent' : 'var(--color-dark)')};
   }
 
   &::placeholder {
     color: 'var(--color-grey-6)';
   }
 
   @media screen and (min-width: 768px) {
-    background-color: ${({ $backgroundcolor }) => $backgroundcolor || 'var(--color-grey-4)'};
+    background-color: ${({ $backgroundcolor }) =>
+      $backgroundcolor || 'var(--color-grey-4)'};
   }
 `;
 
 export const ErrorStyled = styled.p`
   ${generalNotificationStyles}
   color: var(--color-error);
+  position: absolute;
+  top: 58px;
 `;
 
 export const SuccessStyled = styled.p`
   ${generalNotificationStyles}
   color: var(--color-success);
 `;
 
-export const PasswordReapetLable = styled.p`
-  ${generalNotificationStyles}
-  padding-left: 14px;
-  color: 'var(--color-grey-6)';
+export const SuccessIcon = styled(AiOutlineCheck)`
+  width: 20px;
+  height: 20px;
+  color: var(--color-success);
+  position: absolute;
+  right: 50px;
+  top: 18px;
 `;
 
+// export const PasswordReapetLable = styled.p`
```

Comment:
If we don't need this code anymore, let's remove it.

Reply:
Done